### ISSUE TITLE
[FATAL ERROR] the app is totally unusable

### ISSUE THREAD

**Issue by eliasfinesoft** [itsaky/AndroidIDE#341]
Hi itsaky. I appreciate the time you've spent developing AndroidIDE.
But there is a problem that you must solve as soon as possible. Steps to reproduce the problem:

1. install AndroidIDE v2.1.1-beta from Github releases.
2. Open the app and grant storage permission.
3. "CREATE NEW PROJECT" (from the app).
4. After the new project has been successfully created, the EditorActivity is started.
5. There you will see that the red progress indicator will never stop moving, and the following is displayed in the "IDE logs" tab.

```
GradleBuildService        I   Showing notification to user... 
EditorActivity            I   Gradle build service has been started... 
FileTreeFragment          E   Unable to restore tree state  treeState=null  treeView=com.unnamed.b.atv.view.AndroidTreeView@dc75e69 
FileTreeFragment          E   Unable to restore tree state  treeState=null  treeView=com.unnamed.b.atv.view.AndroidTreeView@dc75e69 
GradleBuildService        I   Starting tooling API server... 
GradleBuildService        E   Unable to start tooling API server 
GradleBuildService        E  java.io.IOException: error=2, No such file or directory
GradleBuildService        E  	at java.lang.UNIXProcess.forkAndExec(Native Method)
GradleBuildService        E  	at java.lang.UNIXProcess.<init>(UNIXProcess.java:133)
GradleBuildService        E  	at java.lang.ProcessImpl.start(ProcessImpl.java:141)
GradleBuildService        E  	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
GradleBuildService        E   Caused by: java.io.IOException: Cannot run program "/data/data/com.itsaky.androidide/files/usr/opt/openjdk/bin/java" (in directory "/data/data/com.itsaky.androidide/files/home"): error=2, No such file or directory
GradleBuildService        E  	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1050)
GradleBuildService        E  	at com.itsaky.androidide.shell.CommonProcessExecutor.newProcess(CommonProcessExecutor.java:85)
GradleBuildService        E  	at com.itsaky.androidide.shell.CommonProcessExecutor.newProcess(CommonProcessExecutor.java:74)
GradleBuildService        E  	at com.itsaky.androidide.shell.CommonProcessExecutor.execAsync(CommonProcessExecutor.java:65)
GradleBuildService        E  	at com.itsaky.androidide.shell.CommonProcessExecutor.execAsync(CommonProcessExecutor.java:54)
GradleBuildService        E  	at com.itsaky.androidide.services.GradleBuildService$ToolingServerRunner.run(GradleBuildService.java:473)
GradleBuildService        E  	at java.lang.Thread.run(Thread.java:919)
GradleBuildService        E  
GradleBuildService        E  
FileTreeFragment          E   Unable to restore tree state  treeState=  treeView=com.unnamed.b.atv.view.AndroidTreeView@6375176 
```

![Screenshot_2022-08-01-00-16-52-531_com itsaky androidide](https://user-images.githubusercontent.com/40068066/182071319-4db765da-e95a-4ac1-865e-bfbbed86d372.jpg)

**Comment by itsaky** [comment]
Please follow the installation instructions [here](https://github.com/itsaky/androidide-build-tools) to install the required build tools.

**Comment by eliasfinesoft** [comment]
I did everything the installation guide says, and after the whole procedure was successful. Gradle repository was downloaded. And finally AndroidIDE occupies a total space of 1.63GB in internal storage (which is too much for me).
But I still haven't managed to make AndroidIDE successfully build a project as basic as a "Hello World!".
Finally I decided to uninstall AndroidIDE, and I will continue using the old "AIDE", in addition to the recently developed CodeAssist, they work correctly and without occupying 1.63GB!
